### ISSUE TITLE
revdep-rebuild loop with dev-libs/libffi-3.3_rc0

### ISSUE THREAD

**Issue by realnc** [stefantalpalaru/gentoo-overlay#29]
There's an infinite revdep-rebuild loop due to dev-libs/libffi-3.3_rc0 when using vmware-workstation with the bundled-libs USE flag:

```
!!! existing preserved libs:
>>> package: dev-libs/libffi-3.3_rc0
 *  - /usr/lib64/libffi.so.6
 *  - /usr/lib64/libffi.so.6.0.4
 *      used by /opt/vmware/lib/vmware/lib/libgobject-2.0.so.0/libgobject-2.0.so.0 (app-emulation/vmware-workstation-15.0.2.10952284)
 *      used by /opt/vmware/lib/vmware/lib/libp11-kit.so.0/libp11-kit.so.0 (app-emulation/vmware-workstation-15.0.2.10952284)
Use emerge @preserved-rebuild to rebuild packages using these libraries
```

**Comment by realnc** [comment]
I don't actually use bundled-libs anymore. I had it set for a while because vmware wasn't working without it a while ago, and then forgot about it. I disabled it after I got the revdep-rebuild issue, and it seems vmware now works fine without it. But I reported the revdep-rebuild issue regardless.

> By the way, revdep-rebuild and "emerge @preserved-rebuild" are different mechanisms for doing the same thing, with the latter replacing the former in most cases.

I actually meant to say preserved-rebuld rather than revdep-rebuild.
